go.mod

```
@@ -7,6 +7,8 @@ module github.com/ava-labs/avalanchego
 // go.mod (here, only major.minor can be specified)
 go 1.22
```

```suggestion
go 1.22.0
```
builds are failing due to a toolchain issue, which I believe is solved by this change